frontend/src/views/SQR/Form.vue

```diff
@@ -143,117 +94,12 @@ enum SubmissionsBranch {
 
 @Component({
   name: "Form",
-  components: { SQRField, UnwantedParts, SoundIssues, Duration, TextArea },
+  components: { Fields, CancelListItem },
```

Comment:
What is the rationale behind extracting Fields to a separate component? I guess it is used only once, here.

Reply:
It's easier to read as a separate component.

Reply:
OK then.